haas/ext/switches/migrations/brocade/5a6db7a7222d_added_brocade_driver.py

```diff
@@ -0,0 +1,31 @@
+"""Added Brocade driver
+
+Revision ID: 5a6db7a7222d
+Revises: 6a8c19565060
+Create Date: 2016-04-11 16:26:40.715332
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '5a6db7a7222d'
+down_revision = '6a8c19565060'
```

Comment:
Shouldn't this be `None`? Same goes for the corresponding bit of the docstring above.